src/Command/DeleteClientCommand.php

```diff
@@ -0,0 +1,37 @@
+<?php
+namespace App\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\HttpClient\HttpClient;
+use Symfony\Component\Console\Input\InputArgument;
+
+class DeleteClientCommand extends Command
+{
+    protected function configure()
+    {
+        $this
+        ->setName('client:delete')
+        ->setDescription('Delete a client')
+        ->addArgument('username', InputArgument::REQUIRED, "Username for authentication")
+        ->addArgument('password', InputArgument::REQUIRED, "Password for authentication")
+        ->addArgument('id', InputArgument::REQUIRED, "Client's id")
+        ;
+    }
+    
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $httpClient = HttpClient::create();
+        $username = $input->getArgument('username');
+        $password = $input->getArgument('password');
+        $id = $input->getArgument('id');
+        $output->writeln("Delete client ".$id);
+        $response = $httpClient->request('DELETE', 'http://127.0.0.1/api/clients/'.$id, ['auth_basic' => [$username, $password],]);
```

Comment:
API URL should be configurable, with a localhost default. We should be able to use CLI command from a remote host. (applies to other commands that have an embedded URL)

Reply:
I have added a parameter to configure the url, but at the moment only works for bin/console. I am having trouble to find a way to make dependency injection work for the symfony console application

Reply:
This should be a parameter like user and password, I don't understand this dependency injection thing...

Reply:
ok, I was thinking as a configuration variable. I will make the change